package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "fin_cap",
+  "version": "1.0.0",
+  "description": "Dependencies for FinCap site",
+  "main": "application.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "chai": "^4.1.2",
+    "karma": "^2.0.0",
+    "karma-chai": "^0.1.0",
+    "karma-cli": "^1.0.1",
+    "karma-mocha": "^1.3.0",
+    "karma-phantomjs-launcher": "^1.0.4",
+    "karma-requirejs": "^1.1.0",
+    "karma-sinon": "^1.0.5",
+    "mocha": "^5.0.1",
+    "requirejs": "^2.3.5",
+    "sinon": "^4.4.2"
+  },
+  "scripts": {
+    "test": "./test.sh"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moneyadviceservice/fin_cap.git"
+  },
+  "author": "David Trussler",
```

Comment:
should the author be 'MAS Development Team'

Reply:
I have no problem with that! (Added my name more or less automatically when prompted during the build process).